pybikes/data/callabike.json

```diff
@@ -3,10 +3,10 @@
         {
             "tag": "callabike-aschaffenburg",
             "meta": {
-                "latitude": 49.436467,
```

Comment:
To be honest: I don't know. I took a look at the data in request that is made, when a city is chosen. These coordinates are the values from the request. e.g.
{"method":"Map.listBikes","params":[{"lat":"52.519444","long":"13.406667","maxItems":100,"radius":100000}],"id":1492592072925} for Berlin